composer.json

```diff
@@ -27,7 +27,8 @@
         "php": ">=5.4",
         "symfony/console": "~2.8|~3.0|~4.0",
         "symfony/config": "~2.8|~3.0|~4.0",
-        "symfony/yaml": "~2.8|~3.0|~4.0"
+        "symfony/yaml": "~2.8|~3.0|~4.0",
+        "vlucas/phpdotenv": "^2.4"
```

Comment:
The cakephp skeleton [app](https://github.com/cakephp/app/blob/master/composer.json#L12) uses `josegonzalez/dotenv` for providing dotenv support. While phinx is standalone lib it would be better if projects under the cakephp family used same libs when possible.

Reply:
Good idea. I didn't know that. I'll change this to use the `josegonzalez/dotenv` library soon today.

Reply:
I made this change. Went with "2.*" to be consistent with cakephp.